app/assets/stylesheets/admin/forms/_project_step_form.scss

```diff
@@ -1,49 +1,7 @@
 .project-step-form {
-  .error_notification {
-    display: none;
+  #project_step_scheduled_duration_days {
```

Comment:
We can get rid of this file altogether and this can be moved to a different stylesheet, if desired.